### ISSUE TITLE
Running nuclei with multiple templates results in error

### ISSUE THREAD

**Issue by DeC1pher** [projectdiscovery/nuclei#123]
Hello Team,

it looks like the current version (1.1.7) has a issue when running with multiple templates:

`nuclei -l ~/urllist.txt -t ~/nuclei-templates/cves/`

results in:

```
[ERR] Could not parse file '/root/nuclei-templates/cves'
[ERR] Could not parse file '/root/nuclei-templates/cves'
[ERR] Could not parse file '/root/nuclei-templates/cves'
[ERR] Could not parse file '/root/nuclei-templates/cves'
[ERR] Could not parse file '/root/nuclei-templates/cves'
[ERR] Could not parse file '/root/nuclei-templates/cves'
[ERR] Could not parse file '/root/nuclei-templates/cves'
[ERR] Could not parse file '/root/nuclei-templates/cves'
[ERR] Could not parse file '/root/nuclei-templates/cves'
[ERR] Could not parse file '/root/nuclei-templates/cves'
...
```

This started right after updating `go get -u -v github.com/projectdiscovery/nuclei/cmd/nuclei`

Cheers

**Comment by bauthard** [comment]
Hey @DeC1pher, 

it's too early to run newly added feature, wait for us to add all the details how/what has been added and how to use them, we still have to create a release for newly added features to support and expect a blog with in-depth details about nuclei by end of the day.

**Comment by DeC1pher** [comment]
Thanks for your fast reply! My mistake by adding it to my daily update routine ;)

**Comment by bauthard** [comment]
@DeC1pher I'm keeping it open for tracking purposes.

**Comment by Ice3man543** [comment]
Fixed by #124, thank you for the issue @DeC1pher. This was indeed a bug on our side, on the recent branch.
